internal/proposals/logs/dynamic-backend-configuration/fluentbit-operator.md

```diff
@@ -0,0 +1,82 @@
+* fluent-bit-watcher: goroutine synchronization improvements. #74
+* Add support for plugin alias property in input and output specs. #64.
+
+# Feature comparison matrix
+Feature | FluentBit Operator | Kyma Telemetry Operator
+--- | --- | ---
+License | [Apache 2.0](https://github.com/kubesphere/fluentbit-operator/blob/master/LICENSE)| [Apache 2.0](https://github.com/banzaicloud/logging-operator/blob/master/LICENSE)
+Underlying technology | Fluent Bit, Operator built with Kubebuilder | Fluent Bit for log collection, Operator built with Kubebuilder
+Dynamic configuration | CRDs are directly translatable to Fluent Bit config sections in the most straightforward way. | CRDs are simple structured and support FluentBit syntax, which are then cpoied
+Validation | CRD schema validation (fast and preicse syntax validation, no semantic validation) | Uses FluentBit dryrun feature to validate configuration (more a semantic validation, more hard to translate validation error to the actual config line)
+Config reloading | Custom Fluent Bit image with a bootstrapper that starts a child Fluent Bit process and restarts it if the config changes (intransparent to k8s)| If config changes, configuration will be applied and pods restarted (transparent to k8s)
+Passing sensitive info as Secrets | Not implemented | SecretReferences can be passed. SecretReference is a pointer to a Kubernetes secret that should be provided as environment to Fluent Bit.
+Rollback | Not implemented | Not implemented
+Input, Filter, and Output support | Not all features of inputs, filters, and outputs are supported. But it seems since the acquision they are adding more and more. | All official inputs, filters, and outputs are supported. **But** we do not want that the user is able to create new inputs, creating inputs will be restricted (not implemented until now).
+Customization | Custom Fluent Bit parser plugins supported. Inputs, filters, and outputs are planned to be supported | No custom Fluentd plugins supported, user needs to forward the logs using a custom output to another log processor.
+| Daemonset Management | The Daemonset definition is managed by the operator, incl. potential migration logic | Daemonset will only be restarted but the actual definition is un-managed |
+
+# Chances and Risks
+
+## Chances Fluent-Bit Operator
+* Going with the Fluent-Bit operator will allow Kyma to **save investment** and maintenance. The logging configuration will be changeable at runtime without being forced to touch the fluent-bit system parts (main goal of the story). Furthermore it will maintain the Daemonset and might reduce **maintenace efforts on upgrades**.
+* Furthermore, users will use **well known resource definitions** for the configuration with existing documentation (to be proven) and an existing community behind.
+* The software will be **battle proven** and mostly likely will have a higher quality then a custom solution
```

Comment:
```suggestion
* Going with the Fluent-Bit operator will allow Kyma to **save investment** and maintenance. The logging configuration will be changeable at runtime without being forced to touch the fluent-bit system parts (main goal of the story). Furthermore it will maintain the Daemonset and might **reduce maintenance efforts on upgrades**.
* Furthermore, users will use **well known resource definitions** for the configuration with existing documentation (to be proven) and an existing community behind.
* The software will be **battle proven** and mostly likely will have a higher quality than a custom solution.
```